src/server/api/auth/auth.js

```diff
@@ -5,35 +5,37 @@ const { UserQueryModel } = require('../query-models');
 const userQueryModel = new UserQueryModel();
 
 module.exports = (req, res, next) => {
-  if (req.path === '/login' || req.path === '/login/guest') {
-    next();
-  } else {
+    if (req.path === '/login' || req.path === '/login/guest') {
+        next();
+        return;
```

Comment:
why return here? next() means middleware will pass it on right?

Reply:
just to be safe. I think last time I tried it still went through. or maybe didn't but anyway its just an additional check..